src/store.rs

```diff
@@ -281,4 +287,42 @@ mod tests {
 
         assert_eq!(updated_ticket.status, Status::Done)
     }
+
+    #[test]
+    fn add_comment_to_ticket() {
+        //arrange
+        let mut ticket_store = TicketStore::new();
+        let ticket = generate_and_persist_ticket(&mut ticket_store);
+        //act
+        let result = ticket_store.add_comment_to_ticket(ticket.id, "Test Comment".to_string());
+        //assert
+        assert!(result.is_some());
+        let ticket = ticket_store.get(ticket.id).unwrap();
+        assert_eq!(ticket.comments.len(), 1);
```

Comment:
You can also check that it's the same comment you inserted :D

Reply:
I've updated this although I think after I do more research I could refactor this to be a bit less verbose.